cmd/main.go

```diff
@@ -113,7 +113,8 @@ func InitFlags(fs *pflag.FlagSet) {
 }
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
+	//nolint:gosec // We don't need a cryptographically appropriate random number here
+	rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Can we just remove this line? I don't see where we use random numbers in the project.

Reply:
Removed it completely